pkg/infra/pulumi_aws/index.ts.tmpl

```diff
@@ -143,7 +143,7 @@ export = async () => {
 
     {{range $unit := .StaticUnits}}
     staticUnitUrls.push(createStaticS3Website(
-    "{{$unit.Name}}", "{{$unit.IndexDocument}}", "{{$unit.ContentDeliveryNetwork.Id}}", cloudLib
+    "{{$unit.Name}}", "{{$unit.IndexDocument}}", "{{$unit.ContentDeliveryNetwork.Id}}", cloudLib, appName
```

Comment:
is the appname not a part of cloudLib? I thought it was and we could access it that way. If it isnt then we should make it so it is so that all information needed by our IaC code is available in cloudlib.

Reply:
It's not exported. This pattern is used elsewhere though I see a benefit to exporting it as a readonly property.